### PR TITLE
Add boututils

### PR DESCRIPTION
**Pull request by johnomotani**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by johnomotani** [comment]
@ZedThree, @loeiten could you please confirm you're happy to be maintainers of this package by leaving a comment here?

File: recipes/boututils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
+  run:
+    - python >=3.0
+    - numpy
+    - matplotlib
```
Comment (marcelotrevisani) [inline recipes/boututils/meta.yaml:24]:
```suggestion
    - matplotlib-base
```

Comment (johnomotani) [reply recipes/boututils/meta.yaml]: :+1: thanks @marcelotrevisani!

**Comment by ZedThree** [comment]
Yes, happy to be a maintainer!

**Comment by loeiten** [comment]
Yes, happy to be a maintainer!

**Pull request merged**